CI/travis.before_install.sh

```diff
@@ -1,8 +1,4 @@
 #!/bin/bash
-if [ -z "${TRAVIS_OS_NAME}" ] || [ "${TRAVIS_OS_NAME}" = "linux" ]; then
-  echo Before install on linux.
```

Comment:
I don't think there is a reason to keep cruft about. If we need it again, we can get it from the Git history - that's why we use a version control system!